packages/components/src/HtmlRenderer.purs

```diff
@@ -0,0 +1,40 @@
+module KSF.HtmlRenderer where
+
+import Prelude
+
+import Data.Function.Uncurried
+import Data.Maybe (Maybe(..))
+import KSF.HtmlRenderer.Models as Models
+import React.Basic (JSX)
+import React.Basic.DOM as DOM
+import React.Basic.Hooks (Component, component)
+
+foreign import renderHtmlInputImpl          :: Fn1 String JSX
+foreign import renderHtmlInputWithHooksImpl :: Fn2 String (Array Models.JSGenericHook) JSX
+
+renderHtmlInput :: String -> JSX
+renderHtmlInput htmlInput = runFn1 renderHtmlInputImpl htmlInput
+
+renderHtmlInputWithHooks :: String -> Array Models.JSGenericHook -> JSX
+renderHtmlInputWithHooks htmlInput hooks = runFn2 renderHtmlInputWithHooksImpl htmlInput hooks
+
+type Self =
+  { props :: Props
+  }
+
+type Props =
+  { content :: String
+  , hooks   :: Maybe (Array Models.HookRep)
+  }
+
+htmlRendererComponent :: Component Props
```

Comment:
Let's just do this as pure `render` function. A component with no state is really an antipattern with React Hooks, this only adds complexity for no benefit.